Description: Don't write debug files by default if debug is compiled in.
Author: Nick Leverton <nick@leverton.org>

Index: libupnp-1.6.13/upnp/src/api/upnpdebug.c
===================================================================
--- libupnp-1.6.13.orig/upnp/src/api/upnpdebug.c	2011-10-05 13:56:09.000000000 +0100
+++ libupnp-1.6.13/upnp/src/api/upnpdebug.c	2011-10-05 14:13:27.000000000 +0100
@@ -61,10 +61,10 @@
 static FILE *InfoFileHnd = NULL;
 
 /*! Name of the error file */
-static const char *errFileName = strdup("IUpnpErrFile.txt");
+static const char *errFileName = NULL;
 
 /*! Name of the info file */
-static const char *infoFileName = strdup("IUpnpInfoFile.txt");
+static const char *infoFileName = NULL;
 
 int UpnpInitLog(void)
 {
